extensions/applicationinsights-debugplugin-js/sample/README.md

```diff
@@ -0,0 +1,24 @@
+# Sample Site How-To
+
+## What is this?
+
+This is an extremely simple website created to help test and develop the
+debugging plugin. The site itself is just the buttons along the
+top-left corner of the webpage. Clicking on one of these will fire
+its respective function.
+
+The plugin provides all the rest of the functionality.
+
+## Setup and Running
+
+Currently this site requires a locally-built version of the plugin.
+
+1. ensure `npm` is installed
+2. clone this repository
+3. `npm install` while in the main directory
```

Comment:
This should just reference "building" the complete repo from the repo root and link to the references in the main ReadMe. I think this is required because of the package.json version listing (see other comments)